lldb/source/Plugins/TypeSystem/Swift/SwiftDemangle.cpp

```diff
@@ -0,0 +1,28 @@
+//===-- SwiftDemangle.h ---------------------------------------*- C++ -*-===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2020 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+#include "SwiftDemangle.h"
+#include "LRUCache.h"
+
+namespace lldb_private {
+namespace swift_demangle {
+
+static LRUCache<SharedDemangledNode> shared_cache{10};
```

Comment:
If you make this into a static local variable in the getter, then the initializer is run the first time it is requested, which will make LLDB faster to start up.